Bottom: 3082f418c539d169891a35e80bbd4bd30d61a7f1
Top:    6420ae80e5394d43ebd4bcfc44bde2fa265ed129
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:27:52 -0700

fixup0-update-scriptinterpreterpython


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 503224f8c89..70e079c1d90 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -447,9 +447,8 @@ class FileHandleTestCase(lldbtest.TestBase):
             self.assertTrue(re.search(r'error:.*lolwut', errors))
             self.assertTrue(re.search(r'zork', errors))
 
-    #FIXME This shouldn't fail for python2 either.
+
     @add_test_categories(['pyapi'])
-    @skipIf(py_version=['<', (3,)])
     def test_replace_stdout(self):
         f = io.StringIO()
         with replace_stdout(f):
